NSS-Day-Cohort-68/rare-api-monty-py-api
# Feature/view tags

Ticket #36 View Tags

Purpose: To add a supported resource of 'tags' to the json-server file's do_GET definition to return all the tags and add the get_all_tags function to the __init__.py file. 

Testing
1. Run the API debugger and open Postman.
2. Send the GET request http://localhost:3000/tags
3. You should see a 200 code and all the tags in your database
